### PR TITLE
Turned exercise 019 into practice

### PR DESCRIPTION
**Pull request by divyankachaudhari**
Addresses #2198. Turns [Exercise 019](https://github.com/ocaml/ocaml.org/blob/010/data/exercises/019_rotate_list.md) into practice.

Added four extra test cases:
- Rotate with negative n
- Rotate more than the list length
- Rotate zero places
- Rotate Empty List

### CODE REVIEW

File: practice/019/ex.ml
Comment (cuihtlauac) [inline practice/019/ex.ml:1]: The test with negative length fails when running `dune test --root .`

**Review by cuihtlauac — CHANGES_REQUESTED** [review]

File: practice/019/ex.ml
Comment (divyankachaudhari) [reply practice/019/ex.ml:1]: Fixed.

**Comment by divyankachaudhari** [comment]
@cuihtlauac Could you please review this PR? Thank you.

**Review by cuihtlauac — APPROVED** [review]
Thnaks @divyankachaudhari

**Pull request merged**